ElastixConfig.cmake.in

```diff
@@ -24,4 +24,3 @@ set( ELASTIX_HELP_DIR @ELASTIX_HELP_DIR@ )
 
 # Maintain backwards compatibility by also exporting old-style target information
 set( ELASTIX_ALL_COMPONENT_LIBS @AllComponentLibs@ )
```

Comment:
@kaspermarstal Should this line with ELASTIX_ALL_COMPONENT_LIBS also be removed? But if so, what about @AllComponentLibs@?

Reply:
Everything allcomponentlibs-related should be removed :)

Reply:
Also this one? https://github.com/SuperElastix/elastix/blob/develop/CMakeLists.txt#L562